packages/web-components/src/components/masthead/masthead-menu-button.ts

```diff
@@ -115,6 +117,7 @@ class C4DMastheadMenuButton extends HostListenerMixin(CDSHeaderMenuButton) {
   private _renderSentinel = (side: String) => {
     return html`
       <button
+        part="sentinel-button"
```

Comment:
Maybe add a modifier part so we can target the specific button if needed.
```suggestion
        part="sentinel-button sentinel-button--${side}"
```